### PR TITLE
BTA-2455 Update Github documentation about Java SDK updates

### PR DESCRIPTION
**Pull request by zerga9**
https://btcafrica.atlassian.net/browse/BTA-2455

Update Github documentation adding related JAVA SDK changes
- Create and fund endpoint

### CODE REVIEW

File: docs/TransactionsApi.md [hunk 1]
```diff
@@ -411,6 +412,70 @@ apiClient.setBasePath("https://api-sandbox.transferzero.com/v1");
...
+
+ApiClient apiClient = new ApiClient();˜
+apiClient.setApiKey("<key>");
+apiClient.setApiSecret("<secret>");
+apiClient.setBasePath("https://api-sandbox.transferzero.com/v1");
+
+TransactionsApi apiInstance = new TransactionsApi(apiClient);
+TransactionRequest transactionRequest = new TransactionRequest(); // TransactionRequest |
+try {
+TransactionResponse result = apiInstance.createAndFundTransactions(transactionRequest);
```
Comment (aniki21) [inline docs/TransactionsApi.md]: Can we tab this code properly for readability?

**Comment by kvsm** [comment]
Looking at Travis commits, do we actually need this PR? Commit https://github.com/transferzero/transferzero-sdk-java7/commit/2da5be62590471c216470359514712e7c2f8830e seems to update this automatically when the SDK is generated?

**Comment by sztupy** [comment]
you're right this should be automatic, but the fact that we have a PR with >0 files changed means that something is still off

**Comment by kvsm** [comment]
Yes, Travis didn't update the readme, so we should leave that change. The changes to  docs/TransactionsApi.md are already merged so we shouldn't change it again here.

**Comment by zerga9** [comment]
@kvsm @sztupy shall I revert the changes in docs/TransactionsApi.md?

**Comment by kvsm** [comment]
Yes, if you have a look the documentation is already there now that master has been merged in here.

**Pull request merged**